### PR TITLE
refactor category metadata

### PR DESCRIPTION
**Pull request by lsmith77**
I realized that German Umlauts are supported in Anchors after all. Also, this reduces the amount of logic executed at runtime. Finally using the translation Layer for the anchor was always problematic, since the anchors are not locale specific but language specific (see f.e. behavior vs. behaviour).

Note I will now update the HubSpot DB to use the relevant anchors (though will only publish those changes after we have deployed this PR)

### CODE REVIEW

File: tests/test_1_1/test_api/output.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
             "explanation": {
                 "icon": "😟",
                 "text": "Füllwörter machen Text länger und unverständlicher",
-                "url": "https://www.witty.works/de/kategorien/stil#fuellwoerter"
+                "url": "https://www.witty.works/de/kategorien/stil#füllwörter"
```
Comment (lsmith77) [inline tests/test_1_1/test_api/output.json:128]: `füllwörter`

File: tests/test_demo_wordings_german/test_internal_email/output.json [hunk 2]
```diff
@@ -255,7 +255,7 @@
             "explanation": {
                 "icon": "🧊",
                 "text": "Weniger Förmlichkeit schafft mehr emotionale Nähe",
-                "url": "https://www.witty.works/de/kategorien/stil#formalitaet"
+                "url": "https://www.witty.works/de/kategorien/stil#formalität"
```
Comment (lsmith77) [inline tests/test_demo_wordings_german/test_internal_email/output.json:258]: `formalität`

File: tests/test_gender_ending/test_api_gender_ending_article/output.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
             "explanation": {
                 "icon": "✅",
                 "text": "Bestärkt Wert von Diversity, Gleichstellung und Inklusion",
-                "url": "https://www.witty.works/de/kategorien/inklusiv#diversitaet_&_inklusion"
+                "url": "https://www.witty.works/de/kategorien/inklusiv#diversität_&_inklusion"
```
Comment (lsmith77) [inline tests/test_gender_ending/test_api_gender_ending_article/output.json:42]: `diversität_&_inklusion`

File: tests/test_general_cases/test_api/output.json [hunk 4]
```diff
@@ -14,10 +14,10 @@
             "explanation": {
                 "icon": "🚫",
                 "text": " Verletzt und schüchtert ein",
-                "url": "https://www.witty.works/de/kategorien/offene_diskriminierung#beleidigende_und_einschuechternde_sprache"
+                "url": "https://www.witty.works/de/kategorien/offene_diskriminierung#beleidigende_und_einschüchternde_sprache"
```
Comment (lsmith77) [inline tests/test_general_cases/test_api/output.json:17]: `beleidigende_und_einschüchternde_sprache`

File: tests/test_general_cases/test_api_english-behavior-GB/output.json [hunk 5]
```diff
@@ -20,10 +20,10 @@
             "explanation": {
                 "icon": "😒",
                 "text": "Frames authentic behaviour as undesirable and other",
-                "url": "https://www.witty.works/en/categories/biased_language#behaviour"
+                "url": "https://www.witty.works/en/categories/biased_language#behavior"
```
Comment (lsmith77) [inline tests/test_general_cases/test_api_english-behavior-GB/output.json:23]: `behavior`

File: tests/test_general_cases/test_api_gendered_subcategories/output.json [hunk 6]
```diff
@@ -52,7 +52,7 @@
             "explanation": {
                 "icon": "😟",
                 "text": "Bedient altmodisches Konzept von Führung",
-                "url": "https://www.witty.works/de/kategorien/geschlechtsspezifisch#fuehrungsstereotyp"
+                "url": "https://www.witty.works/de/kategorien/geschlechtsspezifisch#führungsstereotyp"
```
Comment (lsmith77) [inline tests/test_general_cases/test_api_gendered_subcategories/output.json:55]: `führungsstereotyp`

File: tests/test_general_cases/test_api_german_gender_acronyms/output.json [hunk 7]
```diff
@@ -26,7 +26,7 @@
             "explanation": {
                 "icon": "😮",
                 "text": "Männer als Standard; non-binäre Menschen ausgeschlossen",
-                "url": "https://www.witty.works/de/kategorien/geschlechtsspezifisch#geschlechter_spezifische_abkuerzung"
+                "url": "https://www.witty.works/de/kategorien/geschlechtsspezifisch#geschlechter_spezifische_abkürzung"
```
Comment (lsmith77) [inline tests/test_general_cases/test_api_german_gender_acronyms/output.json:29]: `geschlechter_spezifische_abkürzung`

File: tests/test_general_cases/test_german_simple_language/output.json [hunk 8]
```diff
@@ -145,7 +145,7 @@
             "explanation": {
                 "icon": "😉",
                 "text": "Werden alle Lesenden diese Abkürzung verstehen?",
-                "url": "https://www.witty.works/de/kategorien/stil#abkuerzung"
+                "url": "https://www.witty.works/de/kategorien/stil#abkürzung"
```
Comment (lsmith77) [inline tests/test_general_cases/test_german_simple_language/output.json:148]: `abkürzung`

File: tests/test_grammatically_correct_alternatives/test_german_boss/output.json [hunk 9]
```diff
@@ -26,7 +26,7 @@
             "explanation": {
                 "icon": "😟",
                 "text": "Bedient altmodisches Konzept von Führung",
-                "url": "https://www.witty.works/de/kategorien/geschlechtsspezifisch#fuehrungsstereotyp"
+                "url": "https://www.witty.works/de/kategorien/geschlechtsspezifisch#führungsstereotyp"
```
Comment (lsmith77) [inline tests/test_grammatically_correct_alternatives/test_german_boss/output.json:29]: `führungsstereotyp`

File: tests/test_abbreviation/test_german_abbr/output.json [hunk 10]
```diff
@@ -49,7 +49,7 @@
                 "url": "https://www.witty.works/de/kategorien/sprachliche_voreingenommenheit#geschlechtsidentitaet"
             },
             "gravity": 2.0,
-            "label": "Sprachliche Voreingenommenheit: Geschlechtsidentität",
+            "label": "Sprachliche Voreingenommenheit: Geschlechtsidentitaet",
             "start": 25,
             "subcategory": "gender_identity",
```
Comment (lnazarenko) [inline tests/test_abbreviation/test_german_abbr/output.json]: Mention that ä were replace to ae here: Geschlechtsidentität -> Geschlechtsidentitaet

File: tests/test_general_cases/test_api_english-behavior-GB/output.json [hunk 11]
```diff
@@ -20,10 +20,10 @@
...
             },
             "gravity": 2.0,
-            "label": "Biased language: Behaviour",
+            "label": "Biased Language: Behaviour",
             "start": 11,
             "subcategory": "behavior",
```
Comment (lnazarenko) [inline tests/test_general_cases/test_api_english-behavior-GB/output.json:28]: Here label consists: “Behaviour” and subcategory “behavior"

**Review by lnazarenko — APPROVED** [review]

**Pull request merged**